op-e2e/setup.go

```diff
@@ -324,14 +330,28 @@ func (cfg SystemConfig) start() (*System, error) {
 		}
 	}
 
+	// Configure connections to L1 and L2 for rollup nodes.
+	// TODO: refactor testing to use in-process rpc connections instead of websockets.
+	for name, rollupCfg := range cfg.Nodes {
+		rollupCfg.L1 = &rollupNode.L1EndpointConfig{
+			L1NodeAddr: l1Node.WSEndpoint(),
+			L1TrustRPC: false,
+		}
+		rollupCfg.L2s = &rollupNode.L2EndpointsConfig{
+			L2EngineAddrs:      []string{sys.nodes[name].WSEndpoint()},
```

Comment:
Is this the line that needs to be changed to the `AuthEndpoint` for this to work?